unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -554,6 +554,11 @@ abstract public class BaseFPSAgentController {
 
         public abstract void InitializeBody();
 
+         private bool ValidRotateStepDegreesWithSnapToGrid(float rotateDegrees) {
+            // float eps = 0.00001f;
```

Comment:
instead of some epsilon, might be worth using `Mathf.approximately` to deal with floats being weird https://docs.unity3d.com/ScriptReference/Mathf.Approximately.html

Reply:
approximately also uses an epsilon to test so I'll keep it as it is.

Reply:
ah yes I see. As is works for me then.